pyprojectutils/cli.py

```diff
@@ -83,6 +85,92 @@ def generate_password():
     sys.exit(EXIT_OK)
 
 
+def hold_project_command():
```

Comment:
I eventually want other commands to follow the pattern of `action` + `object` + `_command()`.